purchase_supplierinfo_product_breakdown/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
```

Comment:
* In vendor, Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.